### PR TITLE
Minutes for coordination meeting #273, Aug. 29

### PR DESCRIPTION
**Pull request by jouvin**
Preliminary version of Aug. 29 minutes. *Do not merge before Sept. 3", the deadline given to all contributors to fill the minutes, as this meeting was not much attended...

### CODE REVIEW

File: organization/_posts/2024/2024-08-29-coordination.md [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+The draft [slides](https://docs.google.com/presentation/d/1361TObGpuKHpOLXwaubsBNdJBYPG5aV_32eYblKjpvI/edit?usp=sharing) are available.
+
+### HSF Seminar Series
+
+We need to progress on organising the Seminar Series for the Autumn. Likely we could have 3 meetings this year at the start of October, November and December? (for September it’s rather late now).
+- Using the Computing & Accelerator forum slot (Wednesday 4:30-6:00 pm Geneva time) on other weeks
+    - Proposal: last Wednesday of the month
+- First seminar foreseen in October: 30th may be in an holiday period
+    - Alternative: 16th (23d is CHEP week)
+    - Will get in touch soon with Sofware Triggers & Reco who proposed something around 4-D reconstruction
```
Comment (graeme-a-stewart) [inline organization/_posts/2024/2024-08-29-coordination.md:80]: Software

File: organization/_posts/2024/2024-08-29-coordination.md [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+---
+
+## Steering Group News
+
+There was a Steering Group meeting on 26 August and the [minutes](https://hepsoftwarefoundation.org/organization/2024/08/26/steering.html) are available on the website.
+
+- Steering group is now the official name on the website (if you spot arachonistic references to the Coordination Team please make a PR)
+- The HSF Affiliated Projects description has been lightly modified ([document](https://docs.google.com/document/d/1Un1V21LdehQNwkNGeUZOl_GBQ8IdjUpDp5bi9g2YvLg/edit?usp=sharing), [guidelines on levels](https://docs.google.com/document/d/1AiLcqyLA4c1y2Iq-YZyKP7DwN8m2AJb_J44cDuGGAXI/edit))
+    - This is now to be circulated to the Forum for feedback (and a call for early adopters)
+- The HSF Advisory Group [document](https://docs.google.com/document/d/1L62pleuuME6K9WLl5T4KQaImPP_Qpc8HSym-XBNTKv8/edit?usp=sharing) recieived a few points of feedback which will be delt with
```
Comment (graeme-a-stewart) [inline organization/_posts/2024/2024-08-29-coordination.md:95]: dealt

File: organization/_posts/2024/2024-08-29-coordination.md [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+Next meeting October 2 at 4:30 pm Geneva time
+- Will have a general Julia Computing Seminar on 1 October
+    - *Could badge this also as the first HSF Seminar?*
+
+## Other Interest and Activity Areas
+
+### Compute Accelerator Forum
+
+- [11 Sep](https://indico.cern.ch/event/1329693/), CARM profiling tool
+- [9 Oct](https://indico.cern.ch/event/1329694/), RISC-V extended presicion
```
Comment (graeme-a-stewart) [inline organization/_posts/2024/2024-08-29-coordination.md:168]: precision

File: organization/_posts/2024/2024-08-29-coordination.md [hunk 4]
```diff
@@ -0,0 +1,196 @@
+---
+title: "HSF Coordination Meeting #273, 29 August 2024"
+layout: plain_toc
+---
+
+# HSF Coordination Meeting #273, 29 August 2024
```
Comment (graeme-a-stewart) [inline organization/_posts/2024/2024-08-29-coordination.md:6]: No need for a title when it's in the YAML metadata

File: organization/_posts/2024/2024-08-29-coordination.md [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
+
+---
```
Comment (graeme-a-stewart) [inline organization/_posts/2024/2024-08-29-coordination.md:8]: Take out lines 8-19, they are for the live document, not the web archive

File: organization/_posts/2024/2024-08-29-coordination.md [hunk 6]
```diff
@@ -0,0 +1,196 @@
...
+    - This is now to be circulated to the Forum for feedback (and a call for early adopters)
+- The HSF Advisory Group [document](https://docs.google.com/document/d/1L62pleuuME6K9WLl5T4KQaImPP_Qpc8HSym-XBNTKv8/edit?usp=sharing) recieived a few points of feedback which will be delt with
+- We propose to remove the difference between WGs and other activities: everythng will now be called activity.
+    - Web site will be updated accordingly
+
+---
+
+## Activity Updates
+
+### General Standing Reminders
```
Comment (graeme-a-stewart) [inline organization/_posts/2024/2024-08-29-coordination.md:103]: Take out L103-124 (live reminders, not for web)

**Review by graeme-a-stewart — CHANGES_REQUESTED** [review]
Thanks a lot @jouvin - there are a few points to tidy up. I was probably responsible for many of the spelling mistakes.

**Comment by jouvin** [comment]
Thanks @graeme-a-stewart . I accepted your MR with all these changes implemented. I'll try not to forget to update the meeting chair document to mention the sections that need to be removed (except I missed them!).

**Comment by jouvin** [comment]
@all last edits from live notes integrated. Minutes are now ready for merging.

File: organization/_posts/2024/2024-08-29-coordination.md [hunk 7]
```diff
@@ -0,0 +1,156 @@
+---
+title: "HSF Coordination Meeting #273, 29 August 2024"
+layout: plain_toc
+---
+
+## Attending
+
+Present/Contributing: Graeme Stewart, Michel Jouvin, Stefan Roiser, Tommaso Lari
```
Comment (graeme-a-stewart) [inline organization/_posts/2024/2024-08-29-coordination.md:8]: I should be taken out of "present" and left below

**Review by graeme-a-stewart — CHANGES_REQUESTED** [review]
Hi Michel - thanks very much for gathering all the additional material! I spotted only now small point that needs changed.

File: organization/_posts/2024/2024-08-29-coordination.md (hunk 7 above)
Comment (jouvin) [reply organization/_posts/2024/2024-08-29-coordination.md]: Done... and Paul added!

**Review by graeme-a-stewart — APPROVED** [review]

**Comment by graeme-a-stewart** [comment]
Thank you Michel! 🙏

**Pull request merged**